src/toil/wdl/wdltoil.py

```diff
@@ -3091,11 +3095,9 @@ def main() -> None:
             # Load the WDL document
             document: WDL.Tree.Document = WDL.load(options.wdl_uri, read_source=toil_read_source)
 
-            if document.workflow is None:
-                # Complain that we need a workflow.
-                # We need the absolute path or URL to raise the error
-                wdl_abspath = options.wdl_uri if not os.path.exists(options.wdl_uri) else os.path.abspath(options.wdl_uri)
-                raise WDL.Error.ValidationError(WDL.Error.SourcePosition(options.wdl_uri, wdl_abspath, 0, 0, 0, 1), "No workflow found in document")
+            # See if we're going to run a workflow or a task
+            # MiniWDL will handle all the checks
+            target: Union[WDL.Tree.Workflow, WDL.Tree.Task] = runner_exe(document)  # type: ignore[no-untyped-call]
```

Comment:
It looks like this function is part of miniwdl's CLI and expectes to be able to tell the user [to use a `--task` option](https://github.com/chanzuckerberg/miniwdl/blob/a34aa902ec1be2f411db1e38daf51bae4f839f42/WDL/CLI.py#L1101-L1104) if the file has multiple tasks and no workflow. Either we need to make it not do that here or we need to actually add and document a working `--task` option.